team_production_system/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+from .availability import Availability
```

Comment:
Let's add ` # flake8: noqa`  as the top line of this file so the flake8 linter ignores it. We might eventually want to set up a flake8 configuration file but I think we can postpone that for now.